src-frontend/app/components/page-list.js

```diff
@@ -10,13 +10,19 @@ export default Ember.Component.extend({
             scroll: true,
             placeholder: 'ui-state-highlight',
             update: function(event, ui) {
-                var indices = {};
+                var sortOrder = Ember.$(this).sortable('toArray');
+                var pageModels = [];
 
-                Ember.$(this).find('a.open-page').each(function(index) {
-                    indices[Ember.$(this).data('id')] = index;
+                sortOrder.forEach(function(id, index) {
+                    pageModels.push({
+                        id: id,
+                        display_index: index
+                    });
                 });
 
-                pageListComponent.sendAction('updateSortOrder', indices);
+                Ember.$(this).sortable('cancel');
```

Comment:
can move this to the top of this method since it's unrelated to the model update